aten/src/ATen/native/MaxUnpooling.cpp

```diff
@@ -11,6 +11,10 @@ Tensor& max_unpooling2d_forward_out_cpu(
     const Tensor& indices_,
     IntArrayRef output_size,
     Tensor& output) {
+  // See Note [Writing Nondeterministic Operations]
+  // Nondeterministic with duplicate indices
+  at::globalContext().alertNotDeterministic("max_unpooling2d_forward_out");
```

Comment:
Am I missing something? But shouldn't we have these alerts enabled for 1D case as well?